### PR TITLE
replace server side timezone querying logic

### PR DESCRIPTION
**Pull request by georgehrke**
fixes https://github.com/nextcloud/calendar/issues/82
fixes https://github.com/nextcloud/calendar/issues/83

### CODE REVIEW

**Comment by georgehrke** [comment]
ToDo: 
- [ ] add proper readme to `js/timezones` about license MPL v2
- [ ] update zones.json to latest 2017b

File: js/package.json [hunk 1]
```diff
@@ -49,5 +49,8 @@
   },
   "engines": {
     "node": ">=6"
+  },
+  "dependencies": {
+    "gulp-insert": "^0.5.0"
```
Comment (tcitworld) [inline js/package.json]: Why not in devdependencies ?

Comment (georgehrke) [inline js/package.json]: Sorry, that was not intended. Will fix it later today

Comment (georgehrke) [inline js/package.json]: done :)

**Pull request merged**